mon-pix/app/components/recover-account-confirmation-step.hbs

```diff
@@ -0,0 +1,40 @@
+<h1>Bonne nouvelle {{@studentInformationForAccountRecovery.firstName}} !</h1>
+<p>Nous avons retrouvé votre compte :</p>
+<p>Si vous constatez une erreur ou si ces données ne sont pas les votres, contactez le support.</p>
+
+<ul>
+  <li for="account-recovery-last-name">
```

Comment:
Question : pourquoi l'attribut `for` ?

Reply:
Ah c'est un vestige de qui avait avant je corrige !